config/routes.yaml

```diff
@@ -4,6 +4,16 @@ controllers:
         namespace: App\Controller
     type: attribute
 
+robots_txt:
+    path: /robots.txt
+    controller: Symfony\Bundle\FrameworkBundle\Controller\TemplateController
+    defaults:
+        template: robots.txt.twig
+        private: false
+        maxAge: 240
+        sharedAge: 480
+        _requestAnalyzer: false
```

Comment:
Only required to not error if a domain/sitemap.xml is called which is not part of a webspace configuration.